src/triage/component/audition/__init__.py

```diff
@@ -192,6 +200,12 @@ def update_metric_filters(self, new_filters, plot=True):
             plot (boolean, default True) Whether or not to also plot model group performance
                 and thresholding details at this time.
         """
+        if new_filters is None:
```

Comment:
I like this functionality, but I think trying to stick the existing functionality and the 'set one filter' functionality in the same function, it becomes confusing.  If, say, the user passes both a list of `new_filters` and all the new arguments, what does it do? It ignores all the new argument and just replaces the filters.

My instinct is to put the new functionality in a new function that just calls update_metric_filters.